scripts/copy-files.js

```diff
@@ -15,6 +16,12 @@ function copyFile(file) {
   }).then(() => console.log(`Copied ${file} to ${buildPath}`));
 }
 
+function copyTypings(from, to) {
```

Comment:
@oliviertassinari I had to update this script, because the `.d.ts` were not picked up while building :(